js/services/Poll.js

```diff
@@ -14,23 +14,30 @@ define(['knockout', 'visibilityjs'], (ko, Visibility) => {
 
   class PollService {
     add(callback, interval = 1000, ...args) {
-      const intervalId = setInterval(() => {
         if (isPageForeground()) {
```

Comment:
So how the polling will be continued after you first switched from the tab somewhere and then reached it back?